site/docs/about/roadmap.mdx

```diff
@@ -94,3 +43,23 @@ import LargeParagraph from "../../src/components/LargeParagraph";
 
 
 - **HDS 1.0 Release**
+
+**Q2 2021**
+- Image usage instructions
+- Tone of voice instructions
+- Basic tables
```

Comment:
These are formatted differently than other sections. 

For example; "Basic tables" vs "Component: Basic tables".